src/app/search-view/search-view.component.ts

```diff
@@ -12,12 +14,14 @@ import { ApiService } from '../services/api.service';
   styleUrls: ['./search-view.component.css'],
 })
 export class SearchViewComponent {
-  constructor(private apiService: ApiService) {}
+  $dataSearch: Observable<TvShow[]>;
```

Comment:
For observables, the convention is to use $ as a suffix, not prefix, though that doesn't really matter.